src/SfxWeb/src/app/modules/backup-restore/storage-form/storage-form.component.html

```diff
@@ -38,24 +38,32 @@
             <dd>
                 <input type="text" class="input-flat" formControlName="Path" name="Path">
             </dd>
+            <dt>IsPrimaryCredsEmpty</dt>
+            <dd class="checkbox-container-default">
+                <input type="checkbox" formControlName="IsEmptyPrimaryCredential"/>
+            </dd>
             <dt>PrimaryUserName</dt>
             <dd>
-                <input type="text" class="input-flat" formControlName="PrimaryUserName" name="PrimaryUserName">
+                <input type="text" class="input-flat" formControlName="PrimaryUserName" name="PrimaryUserName" [attr.disabled]="localForm.value.IsEmptyPrimaryCredential?true:null">
```

Comment:
dont use attr.disabled for setting the input to disabled. set the form control to disabled in the controller. This applies for the following below.

Reply:
Thanks. I have made the necessary changes